source/drivers/php.txt

```diff
@@ -17,44 +17,56 @@ MongoDB PHP Driver
 Introduction
 ------------
 
-The currently maintained driver is the `mongodb
-<https://pecl.php.net/package/mongodb>`_ extension available from PECL.
+The PHP driver consists of two components, the MongoDB `extension <https://github.com/mongodb/mongo-php-driver>`_
+and `library <https://docs.mongodb.com/php-library/current/>`_.
 
-You might consider using the driver with the complimentary `PHP library
-<https://github.com/mongodb/mongo-php-library>`_, which implements a more
-full-featured API on top of the bare-bones driver.
+The extension provides a low-level API and mainly serves to integrate
+`libmongoc and libbson <https://docs.mongodb.com/ecosystem/drivers/c/>`_ with PHP.
 
-Further information on this architecture may be found 
-in the `PHP.net documentation <http://php.net/manual/en/mongodb.overview.php>`_.
+While it is possible to use the extension alone, users are strongly encouraged
+to use the extension and library together. The library provides a high-level API
+consistent with other MongoDB language drivers.
 
-- `Tutorials <https://www.php.net/manual/en/mongodb.tutorial.php>`__
+- `Tutorials <https://docs.mongodb.com/php-library/current/tutorial>`__
 
-- `Driver Architecture and Internals <https://www.php.net/manual/en/mongodb.architecture.php>`__
+- `Library Architecture <https://www.php.net/manual/en/mongodb.architecture.php>`__
 
-- `API Reference <https://php.net/manual/en/book.mongodb.php>`_
+- API Reference
 
-- `Changelog <https://github.com/mongodb/mongo-php-driver/releases>`__
+  - `Library <https://www.php.net/manual/en/set.mongodb.php>`__
 
-- `Source Code <https://github.com/mongodb/mongo-php-driver>`__
+  - `Extension <https://php.net/manual/en/book.mongodb.php>`__
+
+- Changelog
+
+  - `Library <https://github.com/mongodb/mongo-php-library/releases>`__
+
+  - `Extension <https://github.com/mongodb/mongo-php-driver/releases>`__
+
+- Source Code
+
+  - `Library <https://github.com/mongodb/mongo-php-library>`__
+
+  - `Extension <https://github.com/mongodb/mongo-php-driver>`__
 
 
 Installation
 ------------
 
-You can install the driver using 
-`PECL <https://www.php.net/manual/en/mongodb.installation.pecl.php>`_:
+The preferred method of installing the PHP library is with
+`Composer <https://getcomposer.org/>`_ by running the following from your project root:
```

Comment:
I appreciate the sentiment of instructing users to install the library up front; however, if they actually try to run these steps in the order presented this will fail since the library requires the extension to already be installed.

It may make sense to revert to showing the extension installation step first.